webapp/package.json

```diff
@@ -29,7 +29,7 @@
     "react-dom": "15.4.2",
     "react-intl": "2.2.3",
     "react-router": "2.8.1",
-    "react-select": "1.0.0-rc.3",
+    "react-select": "1.0.0-rc.2",
```

Comment:
It exists, the build would fail if it didn't. I noticed that it was missing from their GitHub repo before but it's on npm